go/testdata/sakila-schema-info.json

```diff
@@ -0,0 +1,70 @@
+{
+  "FileType": "dbinfo",
```

Comment:
According to https://google.github.io/styleguide/jsoncstyleguide.xml?showone=Property_Name_Format#Property_Name_Format:

> Property names must be camel-cased